api/src/main/resources/messages.properties

```diff
@@ -643,6 +643,13 @@ pihcore.visitNote.plan.approximateReturnVisitDate=Approximate next appointment d
 pihcore.visitNote.ncdInitial=NCD 
 pihcore.visitNote.ncdFollowup=NCD
 
+pihcore.visitNote.referredBy=Referred by
+pihcore.visitNote.hasAppointment=Has appointment
+pihcore.visitNote.walkIn=Walk-in
+pihcore.visitNote.activeCasefinding=Active casefinding
+pihcore.visitNote.chw=CHW
```

Comment:
This is fine.  One suggestion -- no need to add visitNote to the message string -- especially not for CHW.  If it's a commonly used string, just use pihcore.chw=CHW